src/modules/field/field.js

```diff
@@ -583,6 +583,9 @@ function options_field_widget_form(form, form_state, field, instance, langcode,
                 if (items[delta].required) {
                   delete items[delta].options[''];
                 }
+                if (items[delta].item && typeof items[delta].item.value !== 'undefined') {
+                  items[delta].value = items[delta].value = items[delta].item.value;
```

Comment:
The extra equals sign in either `===` or `!==` is to add type checking to the comparison.  In this case, however, it's unnecessary because `typeof` always returns a string.